src/desktop/ApplicationWindow.js

```diff
@@ -122,6 +122,7 @@ export class ApplicationWindow {
 				// the preload script changes to the web app
 				contextIsolation: false,
 				webSecurity: true,
+				enableRemoteModule: false,
```

Comment:
Is this related to the zooming? I'm okay with including it but maybe in a separate commit or at least commit should say why it's here.

Reply:
it's not related. It is split into its own commit now.

Reply:
Thanks! Shouldn't affect us, right? I think we don't use it anymore.

Reply:
we removed remote usages a while ago and as of electron 9 there's even a deprecation warning if the module is used without explicitly enabling it in the webPreferences, which we didn't do. this change shouldn't affect us but removes the possibility of any rogue code messing with the module.